Sofa/Component/SolidMechanics/FEM/Elastic/src/sofa/component/solidmechanics/fem/elastic/BaseLinearElasticityFEMForceField.inl

```diff
@@ -82,10 +126,68 @@ auto BaseLinearElasticityFEMForceField<DataTypes>::getYoungModulusInElement(sofa
     }
     else
     {
-        setYoungModulus(5000);
+        setYoungModulus(defaultYoungModulusValue);
         youngModulusElement = youngModulus[0];
     }
     return youngModulusElement;
 }
 
+template <class DataTypes>
+typename BaseLinearElasticityFEMForceField<DataTypes>::Real
+BaseLinearElasticityFEMForceField<DataTypes>::getPoissonRatioInElement(
+    sofa::Size elementId)
+{
+    Real poissonRationElement {};
+
+    const auto& poissonRatio = d_poissonRatio.getValue();
+    if (poissonRatio.size() > elementId)
+    {
+        poissonRationElement = poissonRatio[elementId];
+    }
+    else if (!poissonRatio.empty())
+    {
+        poissonRationElement = poissonRatio[0];
+    }
+    else
+    {
+        setPoissonRatio(defaultPoissonRatioValue);
+        poissonRationElement = poissonRatio[0];
+    }
```

Comment:
The original code did that. The intention is to define only one value that is repeated for the whole vector. Otherwise the user would have to define `youngModulus = "5000 5000 5000 5000 5000 ..."`, a list of nbTetra elements. Or we can resize at init or every topological change.